doc/TESTS.rst.txt

```diff
@@ -370,7 +384,15 @@ failures without requiring a fixed seed, reporting *minimal* examples for
 each failure, and better-than-naive-random techniques for triggering bugs.
 
 
+.. _numpy.test:
```

Comment:
```suggestion
```
This clashes with the internal reference generated by the `.. autofunction` below.